backend/substrapp/tasks/tasks.py

```diff
@@ -678,31 +678,46 @@ def _do_task(client, subtuple_directory, tuple_type, subtuple, compute_plan_id,
             secret_namespace = os.getenv('K8S_SECRET_NAMESPACE', 'default')
             label_selector = f'compute_plan={compute_plan_tag}'
 
+            # fetch secrets and write them to disk
             k8s_client = get_k8s_client()
             try:
                 secrets = k8s_client.list_namespaced_secret(secret_namespace, label_selector=label_selector)
-            except ApiException as e:
+            except kubernetes.client.ApiException as e:
```

Comment:
`from kubernetes.client.rest import ApiException` to `kubernetes.client.ApiException` ?

Reply:
because the `kubernetes.client` package is also used to create the Secret structure (introduced in this PR).

Reply:
Good catch! Will be updated.